### DISCUSSION TITLE
Lower playback speed results in lower pitch

### DISCUSSION THREAD

**Discussion by dKalKul** [alnitak/flutter_soloud#79] [Q&A] [answered]
Hi there.

is there any option to change playback speed to 0.9x or 0.8x without lowering the playback pitch at the same time?

thank you for your answers

**Comment by alnitak** [comment]
Please, take a look at the doc [here](https://pub.dev/documentation/flutter_soloud/latest/flutter_soloud/SoLoud/setRelativePlaySpeed.html)

**Reply by dKalKul** [reply]
Ok, does that mean SoLoud doesn't support changing speed without automatically changing pitch?
Is it planned for future?

**Reply by alnitak** [reply]
Yes, the pitch filter is planned, but keep in mind that by slowing down the speed it is natural to hear a lower pitch.

**Reply by dKalKul** [reply]
Sorry for the very late response and thanks for your help - So, "pitch filter" means that I can set it to change only the tempo without affecting the pitch?

**Reply by alnitak** [reply, marked as answer]
The pitch filter has been added. You can try it using the `pitch` branch. In the `example/lib/filters` you can find the example which uses the pitch filter and a way to modify the speed without changing the pitch. By looking at that example you will have a clearer idea of how the pitch works.
